pkg/systemd/host.js

```diff
@@ -778,6 +778,7 @@ PageSystemInformationChangeSystime.prototype = {
         $('#systime-time-hours').on('input', enable_apply_button);
         $('#systime-date-input').on('input', enable_apply_button);
         $('#systime-timezones').on('change', enable_apply_button);
+        $('#change_systime').on('click', enable_apply_button);
```

Comment:
![screenshot from 2016-06-29 11-04-41](https://cloud.githubusercontent.com/assets/7456909/16442752/49be490a-3df2-11e6-99e8-6f34d3804279.png)
fixes the bug where change button is disabled, after giving invalid input in manual time.